lib/model/unreads.dart

```diff
@@ -39,13 +39,17 @@ class Unreads extends ChangeNotifier {
     required ChannelStore channelStore,
```

Comment:
`unreads [nfc]: Introduce _reverseStreamsLookup data structure`

The commit summary line needs an update to match the new name.